lib/lightning/metadata_service.ex

```diff
@@ -110,13 +110,17 @@ defmodule Lightning.MetadataService do
   end
 
   defp get_output_path(result) do
-    path =
+    last_message =
       result
       |> CLI.Result.get_messages()
       |> List.last()
 
     cond do
-      path ->
+      is_map(last_message) ->
```

Comment:
There is arguably some room to refactor the cond here, but this fix has already taken a lot of time (due to my ignorance) so I thought I would leave that for another day as I think this PR already complies with the Bot Scout principle :).